tests/api/worlds/pen_mesh.wbt

```diff
@@ -31,7 +31,7 @@ DEF BOARD Solid {
       }
       geometry Mesh {
         url [
-          "webots://projects/default/worlds/meshes/suzanne.obj"
+          "https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/meshes/suzanne.obj"
```

Comment:
I am not sure to understand why this change is needed.
Normally, `webots://` urls are not used in the CI but are converted as `https://raw.githubusercontent.com` ones.
And I think that it works, at least for meshes defined like that, because otherwise the `pen_mesh.wbt` test will not pass if the suzanne mesh is not found.

Reply:
I will inspect again but in #3566 this was not working.